From 1975055980e0c1938aa4b254c5c4fdfa3ca24a7b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?St=C3=A9phane=20Cerveau?= <scerveau@gmail.com>
Date: Fri, 16 Oct 2026 11:19:47 +0200
Subject: [PATCH] desktop: describe the game in GenericName and Comment

Use a generic name instead of repeating Name, align Comment with the
appdata summary and add the LogicGame category.
---
 data/dev.mooday.squared.desktop.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/data/dev.mooday.squared.desktop.in b/data/dev.mooday.squared.desktop.in
index b48ce3a..8a20190 100644
--- a/data/dev.mooday.squared.desktop.in
+++ b/data/dev.mooday.squared.desktop.in
@@ -1,9 +1,9 @@
 [Desktop Entry]
 Name=Squared
-GenericName=Squared
-Comment=A funny game to eat your time
+GenericName=Puzzle Game
+Comment=Fill the square with numbers
 Type=Application
 Exec=@script_name@
 Terminal=false
-Categories=Game;
+Categories=Game;LogicGame;
 Icon=@icon@
-- 
2.48.1

